src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -95,13 +95,13 @@
     private String jurisdictionIds;
 
 
-    static final String SRA_REGULATED_FALSE = "false";
+    private static final String SRA_REGULATED_FALSE = "false";
 
 
     protected ResponseEntity<OrganisationResponse>  createOrganisationFrom(OrganisationCreationRequest organisationCreationRequest) {
 
         organisationCreationRequestValidator.validate(organisationCreationRequest);
-        organisationCreationRequestValidator.validateJurisdictions(organisationCreationRequest.getSuperUser().getJurisdictions(), prdEnumService.getPrdEnumByEnumType(jurisdictionIds));
+        OrganisationCreationRequestValidator.validateJurisdictions(organisationCreationRequest.getSuperUser().getJurisdictions(), prdEnumService.getPrdEnumByEnumType(jurisdictionIds));
```

Comment:
if you are making call to static method static way, then remove instance variable "organisationCreationRequestValidator" if it is not required anywhere.

Reply:
i think it is used but will double check now

Reply:
it seems to be used a lot still